bin/logstash.lib.sh

```diff
@@ -1,4 +1,9 @@
-basedir=$(cd `dirname $0`/..; pwd)
+unset CDPATH
```

Comment:
I 'moved' it from `bin/logstash` to `bin/logstash.lib.sh` to have all env initialization in `bin/logstash.lib.sh`